services/leagues.ts

```diff
@@ -116,14 +116,16 @@ interface CurrentGameweekSummaryData {
 }
 
 export type CurrentGameweekSummaryState =
-  | Init
   | Loading
   | Error
   | CurrentGameweekSummaryData;
 
 export async function getGameweekSummary(
```

Comment:
Denne trenger vel egentlig ikke ha noen return-type lengre, siden den ikke brukes til noe. Kan heller aldri havne i error-state, siden du kun oppdaterer når det går bra. Så tenker enten at du kan fjerne hele error-biten, eller så må typen på listUpdate være `listUpdate : (state: CurrentGameweekSummaryState) => void`

Reply:
Ja, beholdt den så man kan velge om man vil ha alt eller inkrementelt som konsumer. Men kan sikkert legge det til igjen hvis det blir aktuelt, ja.

Reply:
Ye, da tror jeg heller jeg ville splitta i to funksjoner i så fall. Eller beholde det som 1, men da burde listUpdate være optional i så fall

Reply:
Ja, godt poeng med opts! Fikser